config/prod/minidream-R-env-pub-instance-ec2.yaml

```diff
@@ -0,0 +1,26 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v5.yaml
+stack_name: minidream-R-env-pub-instance2 
```

Comment:
would you mind changing this to match the file name?

Reply:
Done.